floodviz/static/js/floodviz_onready.js

```diff
@@ -18,11 +18,12 @@ document.addEventListener('DOMContentLoaded', function (event) {
 	var hydro_options = {
 		'height': FV.hydrograph_dimensions.height,
 		'width': FV.hydrograph_dimensions.width,
+		'display_ids': FV.hydrograph_display_ids,
 		'div_id': '#hydrograph'
 	};
 
 	var map_figure = FV.mapmodule(map_options);
-	var hydro_figure = FV.hydromodule(hydro_options);
+	var hydro_figure = hydromodule(hydro_options);
 
 	// Use frames to link interactions
 	var map_to_hydro = {
```

Comment:
I agree with this refactor. But I think it fits in better with a new ticket that would encapsulate the FV.default_display_id changes that still need to be made in map, for the sake of consistency across these figure modules. I would happily pick this new ticket up and make those changes tomorrow.